CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# CHAOSS Website - Content, Issues, Requests and Edits
+
+The CHAOSS website is a Wordpress site.
```

Comment:
Add here that content is automatically included from this repository.

Reply:
Added